src/components/calcite-action-group/calcite-action-group.tsx

```diff
@@ -9,6 +10,16 @@ import { Component, Host, h, VNode } from "@stencil/core";
  * @slot - A slot for adding a group of `calcite-action`s.
  */
 export class CalciteActionGroup {
+  // --------------------------------------------------------------------------
+  //
+  //  Properties
+  //
+  // --------------------------------------------------------------------------
+  /**
+   * Indicates the horizontal or vertical layout of the component.
+   */
+  @Prop({ reflect: true }) layout: CalciteLayout = "vertical";
```

Comment:
@asangma why are we adding `layout` to the `calcite-action-group`? This would suggest that an `action-pad` could have a group laid out horizontally and another one laid out vertically.

Reply:
I think this is a reasonable configuration with action-pad in horizontal and action-group in vertical.
![image](https://user-images.githubusercontent.com/12503298/81835619-501afb80-94f7-11ea-9431-684f59ff6c66.png)

Reply:
@asangma this change would affect `action-bar` as well.

Reply:
Aight...I'll limit it to the action-pad for now.